src/styles/landing-page.less

```diff
@@ -44,14 +44,19 @@
   background-color: @landing-side-bar-bg;
   color: @color-pf-white;
   @media(min-width: @screen-sm-min) {
-    flex: 0 1 auto;
-    width: 325px;
+    bottom: 0;
+    overflow: hidden;
+    overflow-y: auto;
+    position: fixed;
+    right: 0;
```

Comment:
There are a number of variables that determine how scroll bars are painted:  OS, OS scrollbar preferences, and browser.  For example, Chrome 56 and Firefox 52 on macOS Sierra with the default macOS scroll bar setting (Show scroll bar: Automatically based on mouse or trackpad) correctly paint the scrollbar on top of the sidebar when the content area is scrolled.  Oddly, Safari 10.1 on the same OS and setting does not.  As best I can tell, this is because of the addition of the inner scroll.   I am thinking it's prudent to eliminate the inner scroll for now so that we can keep things moving forward as there is much additional styling work to be done.   Any objections?  cc:  @jwforres, @spadgett